streams/src/main/java/org/apache/kafka/streams/kstream/KGroupedStream.java

```diff
@@ -36,7 +36,10 @@
 
     /**
      * Combine values of this stream by the grouped key into a new instance of ever-updating
-     * {@link KTable}.
+     * {@link KTable}. The resulting {@link KTable} will be materialized in a local state
+     * store with the given store name. Also a changelog topic named "applicationID-storeName-changelog"
```

Comment:
Nit:

> Also a changelog topic named "applicationID-storeName-changelog"

I assume `-changelog` is hardcoded, so perhaps we should clarify that both `applicationID` and `storeName` are variables in this string?

Something like the following perhaps?

> Also a changelog topic named "${applicationID}-${storeName}-changelog" ...